__tests__/app.test.js

```diff
@@ -45,4 +45,26 @@ describe("/api", () => {
                 expect(body.api).toEqual(endpoints);
             });
     });
+});
+
+describe("/api/articles", () => {
+    test("GET:200 should respond with an articles array of article objects, with the correct properties", () => {
+        return request(app)
+            .get("/api/articles")
+            .expect(200)
+            .then(({ body }) => {
+                const { articles } = body;
+                expect(articles).toHaveLength(13);
+                articles.forEach((article) => {
+                    expect(typeof article.author).toBe("string");
+                    expect(typeof article.title).toBe("string");
+                    expect(typeof article.article_id).toBe("number");
+                    expect(typeof article.topic).toBe("string");
+                    expect(typeof article.created_at).toBe("string");
+                    expect(typeof article.votes).toBe("number");
```

Comment:
good test.  Look up expect.any(**insert data type**) as a way of checking data type if a property as an alternative.